controllers/storagecluster/exporter.go

```diff
@@ -709,3 +715,86 @@ func createMetricsExporterRolebindings(ctx context.Context, r *StorageClusterRec
 
 	return nil
 }
+
+func createRookCephClusterRolebindings(ctx context.Context,
+	r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
+	// rookCephMonitorMgrRoleBinding is a cluster rolebinding for monitor mgr
+	rookCephMonitorMgrRoleBinding := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph-monitor-mgr",
+			Namespace: instance.Namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "rook-ceph-monitor-mgr",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind: "ServiceAccount",
+				Name: "rook-ceph-mgr",
+			},
+		},
+	}
+
+	// rookCephMonitorRoleBinding is a cluster rolebinding for rook-ceph monitor
+	rookCephMonitorRoleBinding := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph-monitor",
+			Namespace: instance.Namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "rook-ceph-monitor",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind: "ServiceAccount",
+				Name: "rook-ceph-system",
+			},
+		},
+	}
+
+	var roleBindings = []*rbacv1.ClusterRoleBinding{
+		rookCephMonitorMgrRoleBinding, rookCephMonitorRoleBinding,
+	}
+
+	for _, expectedRoleBinding := range roleBindings {
+		currentClusterRoleBinding := new(rbacv1.ClusterRoleBinding)
+		currentClusterRoleBinding.Name = expectedRoleBinding.Name
+		currentClusterRoleBinding.Namespace = instance.Namespace
```

Comment:
```suggestion
		currentClusterRoleBinding.ObjectMeta = expectedRoleBinding.ObjectMeta
```
Isn't this more clearer?

Reply:
We just need `Name` and `Namespace`  for `CreateOrUpdate()` function (because in the function it will change the passed pointer reference) =)
Rest everything is added/changed in the update function body.
As you have suggested, made the change. Since we have other complicated data structures in the `ObjectMeta`, used `DeepCopyInto()` function.